INSTALL.rst

```diff
@@ -11,6 +11,24 @@ environment.
 
 1. Standard Installation
 2. Development Installation
+3. Linux operating system installation (not recommended)
+
+Warning
+-------
+
+No matter what anybody tells you, do not type::
+
+  sudo pip [anything else]
+
+This is almost guaranteed to not only destroy your installation of python, but
+also your whole operating system since the operating system expects very
+specific versions of libraries to be installed. In the best case, you are left
+with a broken install. In the worst case, your operating system will refuse
+to boot, and the easiest solution might be to perform a clean install.
+
+If you must use the system environment, which we do not recommend, read on
+below.
```

Comment:
We could link here to an external tutorial on Python environments. This just came across the software carpentry mailing list:

https://kaust-vislab.github.io/introduction-to-conda-for-data-scientists/

Reply:
don't most core devs prefer pip?

Reply:
Perhaps. If you see a cool virtualenv tutorial, let me know. ;) Last I checked, venv was not so great for maintaning multiple Python versions, but things might have evolved.

Reply:
I would put this in the sphinx warning box so it is red.